packages/flame/lib/src/events/messages/drag_end_event.dart

```diff
@@ -20,4 +20,7 @@ class DragEndEvent extends Event {
       ),
     );
   }
+
+  @override
+  String toString() => 'DragEndEvent(pointerId:$pointerId, Velocity:$velocity)';
```

Comment:
```suggestion
  String toString() => '$DragEndEvent(pointerId:$pointerId, Velocity:$velocity)';
```
Will help in keeping class name in this string in-sync with the actual class name in future as well.